src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  position: relative;
+}
+
+.input {
+  display: block;
+  font-weight: 300;
+  width: 100%;
+  margin-bottom: 20px;
+  left: 0;
+  top: 0;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: bold;
+}
+
+.bigInput {
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  padding-left: 62px;
+}
```

Comment:
these styles write for .input, delete styles for .bigInput, leave styles for .smallInput
(checklist point 2) - check all similar cases